src/platformsh.js

```diff
@@ -0,0 +1,54 @@
+/**
+* Read number of CPUs from environment or fallback to the _private_ configuration property
+* Useful for determining the number of processes to fork.
+*/
+const num_of_cpus = () => {
```

Comment:
Let's agree to disagree :)

The `const` declaration still has the drawback of hoisting, which can lead to obscure bugs, so I still prefer `function`.